api/configuration_services.proto

```diff
@@ -57,6 +58,89 @@ message GetPhyVersionsResponse {
   repeated VersionInfo version_info = 2;
 }
 
+message ListBandsRequest {
+  // Optional Band ID to filter the results.
+  // If unused, all supported Bands and their versions are returned.
+  string band_id = 1;
+}
+
+message BandDescription {
+  string id = 1;
+
+  message Beacon {
+    DataRateIndex data_rate_index = 1;
+    string coding_rate = 2;
+    bool inverted_polarity = 3;
+    reserved 4; //ComputeFrequency
```

Comment:
Space after `//`, and let's use the converted casing (so `compute_frequency` since that would be field name if we actually filled it. 

Same for the rest of the reserved fields.